### DISCUSSION TITLE
Entrevistas de emprego

### DISCUSSION THREAD

**Discussion by anacsv** [anacsv/tech-br-em-pt#11] [Entrevistas]
Se possivel coloquem se é empresa portuguesa ou de fora
Teve teste tecnico?
O que perguntaram?
Pediram documentos antes?
Teve pré proposta?
Podem falar salario também se quiserem, creio que não tenha problema.
Se não quiserem colocar o nome da empresa podem colocar algum adjetivo, cor da marca, etc.

**Comment by jvitorinoj** [comment]
Cargo: DevOps
Empresa: EPAM
Etapas:

1. Entrevista com RH
2. Entrevista técnica com funcionário da EPAM. Muitas questões sobre arquitetura de infra e cloud.
3. Proposta financeira (isso aqui é estranho, fazem a proposta financeira antes de ter um cliente definido mas o contrato só é feito depois  da aprovação do cliente
4. Varias entrevistas técnicas com vários clientes.

Começaram a forçar a barra me enviando para entrevistas e cargos que eu claramente não tinha perfil. Reclamei e pararam de entrar em contato.

**Comment by jvitorinoj** [comment]
Empresa: CGI
Cargo: DevOps
Etapas:

1. Entrevista com o RH da CGI
2. Entrevista técnica com o cliente. Perguntas conceituais sobre containers, cloud, linux, IaC e CICD
3. Entrevista com o gestor da equipe (cliente).
Mais um processo que caiu no limbo, sem retorno algum.

**Reply by anacsv** [reply]
Muito obrigada por compatilhar conosco

**Comment by fao89** [comment]
Empresa: Red Hat
Cargo: Software Engineer
Etapas:
1. Entrevista com RH (mais para checar o inglês) 
2. Entrevista com gerente (geralmente é p checar soft skills)
3. Geralmente 2 entrevistas técnicas com diferentes pessoas do time

Obs: nas vagas publicadas o que importa mais é a timezone e não o país, então se ver uma vaga "France-remote", pode aplicar!

**Reply by anacsv** [reply]
Muito obrigada por compatilhar conosco

**Comment by Fercs89** [comment]
Cargo: Consultant
Empresa: Closer Consulting (PT)
Etapas:

1. Entrevista com o recrutador
2. Entrevista técnica com um sênior 
3. Proposta
4. Varias entrevistas técnicas com vários clientes.
